apps/src/lib/levelbuilder/rubrics/rubricHelper.js

```diff
@@ -110,7 +110,7 @@ function removeNewIds(keyConceptList) {
 function resetPositionsOfLearningGoals(keyConceptList) {
   let position = 1;
   keyConceptList.forEach(keyConcept => {
-    if (keyConceptList._delete) {
+    if (keyConcept._destroy) {
```

Comment:
is this a typo fix or is there a difference between delete and destroy here?

Reply:
Great question!  A bit of both... I think it was a typo initially.  When we remove a LearningGoal we add a `_destroy` property ([found here](https://github.com/code-dot-org/code-dot-org/blob/2c0066db7cfe18df5c3e4752f45ae04740db5baa/apps/src/lib/levelbuilder/rubrics/RubricsContainer.jsx#L126)). 

I did do [some research](https://medium.com/@wkhearn/delete-vs-destroy-does-it-even-matter-8cb4db6aa660) on `delete` vs `destroy`(because I cannot remember what the reasoning was for using _destroy in the first place), and it seems like we do want `destroy` because we want to remove the associated `evidenceLevels` as well.